2주차/[Lv.2]_[1차]_뉴스_클러스터링/이상훈.js

```diff
@@ -0,0 +1,31 @@
+const makeArr = (str) =>
+  str
+    .split('')
+    .filter((_, idx) => idx !== 0)
+    .map((_, idx) => (str[idx] + str[idx + 1]).toUpperCase())
+    .filter((item) => /[^A-Za-z]/.test(item) === false);
```

Comment:
```suggestion
  str
    .split('')
    .filter((char) => /[A-Za-z]/.test(char))
    .map((char, idx) => (str[idx] + str[idx + 1]).toUpperCase());
```
이렇게 하면 `filter`를 한 번만 사용해서 더 효율적이지 않을까용?👀